tests/basic/pct.rs

```diff
@@ -191,8 +191,8 @@ fn figure1b_pct() {
 fn figure1b_pct_with_many_tasks() {
     // Spawn 48 busy threads, each taking 5 steps, plus 2 main threads with 10 steps, so k=260
     // n=50, k=260, d=2, so probability of finding the bug in one iteration is at least 1/(50*260)
-    // So probability of hitting the bug in 2000 iterations = 1 - (1 - 1/400)^2000 > 99.9%
-    let scheduler = PctScheduler::new(2, 2000);
+    // So probability of hitting the bug in 2000 iterations = 1 - (1 - 1/1300)^60_000 > 99.9%
+    let scheduler = PctScheduler::new(2, 60_000);
```

Comment:
The comment is still out of sync with the code :-) ("2000 iterations", and 50*260 = 13,000)

More importantly, 60,000 iterations is a ton. Maybe we should drop the test to 20 threads (which is still > DEFAULT_INLINE_TASKS). Then the comment math works out as:
```
// Spawn 18 busy threads, each taking 5 steps, plus 2 main threads with 10 steps, so k=110
// n=20, k=110, d=2, so probability of finding the bug in one iteration is at least 1/(20*110)
// So probability of hitting the bug in 16,000 iterations = 1 - (1 - 1/1300)^16_000 > 99.9%
```

Reply:
Fixed :-)